pkg/clusteragent/admission/controllers/secret/controller.go

```diff
@@ -127,7 +127,7 @@ func (c *Controller) handleObject(obj interface{}) {
 
 // handleUpdate handles the new object reported in update events.
 // It can be a callback function for update events.
-func (c *Controller) handleUpdate(oldObj, newObj interface{}) {
+func (c *Controller) handleUpdate(oldObj, newObj interface{}) { //nolint:revive // TODO fix revive unused-parameter
```

Comment:
the parameters comes from an Interface definition.

Reply:
```suggestion
func (c *Controller) handleUpdate(_, newObj interface{}) { //nolint:revive // TODO fix revive unused-parameter
```

Reply:
Note: you can use the github suggestion with <cmd+g> and suggest changes from comments